packages/create-sitecore-jss/src/templates/nextjs-sxa/src/Layout.tsx

```diff
@@ -28,6 +31,26 @@ interface RouteFields {
   Title?: Field;
 }
 
+const DefaultLayout = ({ route }: { route: RouteData<RouteFields> | null }): JSX.Element => (
```

Comment:
Instead of introducing a new component I would recommend to define a function `renderContent` within Layout and just call it
```
const renderMainContent = () => (
   <>
      <header>
      ...
   </>
)
```
It will not be required to pass twice identical props to DefaultLayout
Also, we couldn't really name it as `DefaultLayout` since our wrapper is `LayoutComponent`